### ISSUE TITLE
text changes required in configuration

### ISSUE THREAD

**Issue by ValWood** [PHI-base/canto#9]
@CuzickA  will go through these with @henryosim

**Comment by kimrutherford** [comment]
One change we talked about was the prompt/help text on the gene upload page.  For pombe we have:

>Please list the genes studied in this paper using the systematic identifier (eg. SPCC1739.10) or standard name (eg. cdc11) separated by commas, spaces, tabs or one per line. 

I think we decided that the PHI-Canto users will will need to provide a UniProt ID rather than a gene name.  If that's the case PHI-Canto should say something like:

>Please list the genes studied in this paper using the UniProt identifier (eg. Q00909) separated by commas, spaces, tabs or one per line.

If just fixed the Canto code (in 59a1fce1dd009b001c6116abd0627f9a460355f1) so you can hide the "or standard name (eg. cdc11)" bit.  So @henryosim you'll need to pull the latest Canto code into yours for this to work.

With that Canto fix, if you add the configuration below to `canto_deploy.yaml` it should do the trick:

```
messages:
  gene_upload:
    systematic_identifier_desc: UniProt identifier
    primary_name_desc: ~
    examples:
      gene_identifier: 'Q00909'
      gene_name: ~
```

(The "~" overrides that default value and unsets it)

**Comment by henryosim** [comment]
`messages:
  gene_upload:
    systematic_identifier_desc: UniProt identifier
    primary_name_desc: ~
    examples:
      gene_identifier: 'Q00909'
      gene_name: ~`

has been added to `canto_deploy.yaml` [see here](https://github.com/PHI-base/canto/blob/9026c72a199b224b1706d8dc8e9da57b79f475df/canto_deploy.yaml#L22)
